addons/mrp_account/models/mrp_production.py

```diff
@@ -18,9 +18,16 @@ def _compute_show_valuation(self):
         for order in self:
             order.show_valuation = any(m.state == 'done' for m in order.move_finished_ids)
 
+    def get_sol_analytic_distribution(self):
```

Comment:
```suggestion
    def _get_related_analytic_distribution(self):
```
It's not common to talk about models we don't have access to
That way it makes your method more generic, in case we need to override it somewhere else for another reason
(Also, careful, the method was public)